### ISSUE TITLE
Better progressive prices for host plans

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#3393]
## User story 
As a host admin, I want to pay a reasonable price for my host plan, and not be disincentivised from adding more Collectives because there's a steep price increase at a certain number.

Host admin feedback:

> I’d encourage you to consider a less steep price ramp - going from 5 active collectives to 6 (or from 10 to 11) instantly doubles monthly costs. Also it looks like the bigger and more able the host is to pay the monthly fee, the cheaper those fees become per hosted collective (2 collectives = $12.50 per month per collective; 25 collectives = $2.50 per month per collective).

## Best solution for this problem
Change to a linearly scalable plan like $10/month per active Collective, so instead of the price doubling suddenly it would go up gradually.

If we offered a discount as the number of Collectives rose, it would actually incentivise hosts from onboarding more Collectives, which is what we want. For example, 5% off over 5 Collectives, 10% off over 10 Collectives, up to 50, then beyond 50 we can do custom arrangements.

If we implemented #3392 as well, and only applied the discount and $10/mo charge to *active* Collectives, all the incentives would be aligned. Hosts would onboard more Collectives, and Open Collective would be offering good deals for more active Collectives bringing in money.

**Comment by alanna** [comment]
I did some modelling based on a % discount that smoothly goes up as you add more Collectives, and this is how it turned out.

![Screen Shot 2020-08-06 at 4 34 00 PM](https://user-images.githubusercontent.com/5498878/89491123-9fc14880-d802-11ea-8e29-000a3b1903bd.png)

Progressive pricing is inherently more fair. Look at the weirdness of price per Collective we have currently. However, a big requirement of a pricing scheme is *simplicity*. We struggled so hard to communicate the current pricing plans and anything you need a calculator to figure out seems like too much cognitive overhead.

Then I got to thinking, why do we charge just as much for a Collective with very little money, why don't we just charge a % of overall funds coming in?... and then I realised I was right back where we started years ago. 🤦🏻‍♀️

**Comment by znarf** [comment]
@alanna Well, if you remember I strongly expressed my opinion in the past about this feature.

After 6 months, I think we're now ready to reconsider it.

**Comment by alanna** [comment]
What's your opinion now that we have the last 6 months of experience @znarf ?

It would be useful to have some data about paid host plans, if that's accessible.

**Comment by znarf** [comment]
As of today:
- Single Host Plan: 5 -> $50
- Small Host Plan: 6 -> $150
- Medium Host Plan: 1 -> $50
- Large Host Plan: 1 -> $400
- Total: $600 / monthly

To put in perspective, it's the equivalent of 5% of $12.000. I think it's not worth the friction, we better provide the best service we can and make sure we're helping Hosts grow their revenues.

**Comment by alanna** [comment]
We have kind of done an informal A/B test for host pricing this year, because we gave existing hosts the option to switch to the new pricing or not. I think we can look at some data to see results.

* What % of existing hosts opted to switch to the new pricing vs staying grandfathered in the old model?
* What's the average revenue per host for new pricing plans vs old?
* For hosts on new pricing plans, what is their aggregate revenue vs what it would be on the old pricing plan of 5%?
* What was our rate of onboarding new hosts with the old plan vs the new one? (there are more factors but pricing is an important one - if we landed more new hosts on the new plan than the old it's a sign it's a better offer)
* What's the average rate of adding new Collectives for hosts on the old vs new pricing models? (If they add fewer Collectives on the new plan, or tend to stop just before jumping to the next price bracket, it's a sign they are disincentivised by the pricing increases)

@znarf can you get this data easily?

Revenue and providing good value to our customers is obviously the #1 priority for pricing, but there are other important considerations:

* **Simplicity and communicability**: 
  * People really don't tolerate complex pricing models very well. It was a challenge to communicate the old pricing (split between platform and host), and it was a challenge to communicate the new plans (look at how complex the current pricing page is).
  * For me the biggest source of unwanted complexity wasn't % vs flat $ pricing, it's the host+platform dual pricing message. Making the customer relations with the *host* not the end-user Collective helps a lot. We always struggled to communicate and justify the 5%+5% model, but I think it could be possible to break the pricing relationship between Platform and Collective and only make it between Platform and Host while still basing it on a % of revenue instead of a flat $ rate. However, this means we have to empower hosts to implement their chosen business model to their Collectives (we're on the way to this with new features already, like charging a flat fee and setting fees per Collective), AND find a way to communicate to end-users that their pricing discussion is with their *host* not with us (very hard because people only associate the main Open Collective brand with the whole experience).
  * Other fiscal sponsors often charge a flat rate + a percentage as their pricing model. We considered this but deemed it too complex. We could reconsider?

* **Scalability**:
  * We need host pricing that can grow as a host scales. Some pricing models make sense at small scale but not with big money, and vice versa. We've had to implement variable pricing deals in our largest hosts to accomodate this, and it shows our pricing model is not one size fits all.
  * The comment that kicked off this thread was about meeting the needs of hosts who have a lot of Collectives but aren't bringing in much $. For them, a % of revenue is way better, and it enables people to take the risk to start a host even if revenue isn't guaranteed.
  * At the same time, we found that a flat rate % does not work with big money, which is a big reason we changed it. However, there are other options like reducing the % as money goes up.
  * There's a big difference between the needs of self-hosted Collectives (Fiscal Hosts with one Collective) and multi-Collective hosts. One pricing model doesn't fit both easily.

**Comment by znarf** [comment]
> can you get this data easily?

It's not easy to answer all your questions here. Maybe we can setup a call to work together and find ways to answer these questions. I could easily answer this one:

> What % of existing hosts opted to switch to the new pricing vs staying grandfathered in the old model?

If I look at the 2019 list, only `protozoa` upgraded. The 10 other hosts with an active plan are new Hosts.

**Comment by Upperholme** [comment]
Chipping in on this thread from the perspective of a small host, still learning and feeling our way with the OC platform, and trying to understand how to get the best from it. Our rationale for being a host on OC is because we want to offer s service to small and pre-start/startup cooperative organisations and those that don't want to get bogged down in legal entities etc. As @znarf indicates, not all hosts are the same. I see some hosts that are looking after lots of collectives that are moving a lot of money, and I see others that are more like us, and of course there are many who are simply hosting their own collective. 
We got involved partly because there was no upfront cost to us as a host so we could experiment and learn without having to spend monthly just to be here. We want to encourage new groups that we work with to make use of OC, but if that's going to cost us in terms of a monthly fee from day 1, that acts as a clear disincentive unless we know for sure  that it will bring in the revenue that we're going to need to keep the wagon on the road.
So a smooth ramp in terms of pricing makes sense for us, as does #3392 which would allow us to open up new collectives at no/low risk, and if they don't take off it's not going to cost us. We've got enough to worry about with all the complexities of handling the book-keeping involved (would love to see solutions there), and with international payments in and out in multiple currencies.
